tests/test_transform_probes.py

```diff
@@ -133,6 +133,54 @@
     }
 }
 
+OUT_PROBE_DATA_BY_CHANNEL = {
+    'release': {
+        'histogram/TEST_HISTOGRAM_1': {
+            'history': [
+                {
+                    'cpp_guard': None,
+                    'description': 'A description.',
+                    'details': {
+                        'high': 10,
+                        'keyed': False,
+                        'kind': 'exponential',
+                        'low': 1,
+                        'n_buckets': 5
+                    },
+                    'expiry_version': '53.0',
```

Comment:
While we're here and changing version handling... Let's just strip the output to '53' (i.e. major version only).
We're making that change client-side too and never used minor versions.

Reply:
Done!